.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+on:
+  push:
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
```

Comment:
Can we use version as opposed to what I believe are commit SHAs? Also same comment as stoping CircleCI, disabling it as required, etc. I can do that when you tell me it's ready.

Reply:
Addressed in [`01a06d8` (#73)](https://github.com/panther-labs/panther-auxiliary/pull/73/commits/01a06d8af98d00840df1c2f02bd4633725322ef4). You can disable the CircleCI builds and make the new check required now.